test/test_converter.py

```diff
@@ -42,300 +42,364 @@ def test_populate_endpoints(js_usages_1, js_usages_2):
-                           '/rest/2fa/disable', '/rest/2fa/setup', '/rest/2fa/status',
-                           '/rest/2fa/verify', '/rest/admin/application-configuration',
-                           '/rest/admin/application-version', '/rest/basket', '/rest/basket/{id}',
-                           '/rest/basket/{id}/checkout', '/rest/basket/{id}/coupon/{coupon}',
-                           '/rest/basket/{id}/order', '/rest/captcha', '/rest/chatbot/respond',
-                           '/rest/chatbot/status', '/rest/continue-code',
-                           '/rest/continue-code-findIt',
-                           '/rest/continue-code-findIt/apply/{continueCode}',
-                           '/rest/continue-code-fixIt',
-                           '/rest/continue-code-fixIt/apply/{continueCode}',
-                           '/rest/continue-code/apply/{continueCode}', '/rest/country-mapping',
-                           '/rest/deluxe-membership', '/rest/image-captcha', '/rest/languages',
-                           '/rest/memories', '/rest/order-history', '/rest/order-history/orders',
-                           '/rest/order-history/{id}/delivery-status', '/rest/products/reviews',
-                           '/rest/products/search', '/rest/products/{id}/reviews',
-                           '/rest/repeat-notification', '/rest/saveLoginIp',
-                           '/rest/track-order/{id}', '/rest/user/authentication-details',
-                           '/rest/user/change-password', '/rest/user/data-export',
-                           '/rest/user/login', '/rest/user/reset-password',
-                           '/rest/user/security-question', '/rest/user/whoami',
-                           '/rest/wallet/balance', '/rest/web3/nftMintListen',
-                           '/rest/web3/nftUnlocked', '/rest/web3/submitKey',
-                           '/rest/web3/walletExploitAddress', '/rest/web3/walletNFTVerify',
-                           '/security.txt', '/snippets', '/snippets/fixes',
-                           '/snippets/fixes/{key}', '/snippets/verdict', '/snippets/{challenge}',
-                           '/solve/challenges/server-side', '/support/logs',
-                           '/support/logs/{file}',
-                           '/the/devs/are/so/funny/they/hid/an/easter/egg/within/the/easter/egg',
-                           '/this/page/is/hidden/behind/an/incredibly/high/paywall/that/could/only/be/unlocked/by/sending/1btc/to/us',
-                           '/token', '/trustproxy', '/video', '/viewengine',
-                           '/we/may/also/instruct/you/to/refuse/all/reasonably/necessary/responsibility',
-                           '/x-powered-by', '/{regex_param_1}/{file}', ]
+    result_keys = sorted(result.keys())
+    assert result_keys == ['/',
+    '/.well-known/security.txt',
+    '/123.456.789',
+    '/Blockedillegalactivityby',
+    '/Invalidemail/passwordcannotbeempty',
```

Comment:
These ones got fixed with this atom [PR](https://github.com/AppThreat/atom/pull/133/files#diff-07a425151c151de0154bae7bc1f75b348edb364c1072063b569e11de4c3ce038R562)

Reply:
@prabhu Great! When the PR is merged, I will regenerate the slices we use for testing.